### PR TITLE
OPSCENTER-100: remove * in projections

### PR DESCRIPTION
**Pull request by jinfengni**

### CODE REVIEW

File: bootstrap/016_query_history.sql [hunk 1]
```diff
@@ -57,12 +663,155 @@ SELECT
...
+    qh.QUERY_ACCELERATION_PARTITIONS_SCANNED,
+    qh.QUERY_ACCELERATION_UPPER_LIMIT_SCALE_FACTOR,
+    qh.TRANSACTION_ID,
+    qh.CHILD_QUERIES_WAIT_TIME,
+    qh.ROLE_TYPE
 FROM QH
 LEFT OUTER JOIN INTERNAL_REPORTING.WAREHOUSE_CREDITS_PER_SIZE size ON qh.warehouse_size = size.warehouse_size,
 LATERAL FLATTEN(internal.period_range('hour', qh.start_time, qh.end_time)) emt(index)
 )
 SELECT * EXCLUDE  (PERIOD_PLUS) FROM EXTRAS;
```
Comment (rymurr) [inline bootstrap/016_query_history.sql:736]: do we need to remove this `*` too?

**Review by rymurr — COMMENTED** [review]
I think we are missing the `bootstrap/015_warehouse_views.sql` ones still too

File: bootstrap/016_query_history.sql (hunk 1 above)
Comment (rymurr) [reply bootstrap/016_query_history.sql:736]: you are correct, I missed that

**Comment by jinfengni** [comment]
I pushed a commit which removes * in `015_warehouse_views.sql`.  @rymurr please take another look at this PR. Thanks!

**Review by rymurr — APPROVED** [review]

**Pull request merged**